3-extra/1-count-words.js

```diff
@@ -25,10 +25,19 @@
 
 function countWords(string) {
   const wordCount = {};
-
-  // write code here
-
-  return wordCount;
+  if (string === "") {
+    return wordCount;
+  } else {
+    const myArray = string.split(" ");
```

Comment:
Try to give a meaningful name to the variables like 'myArray'. Something related to what they are doing or their purpose.

Reply:
Changed name of the array . Thanks for review Alvaro !